README.md

```diff
@@ -12,42 +12,40 @@ management, role based access control etc to first and third-party Software as a
 Azure AD Client Connector allows you to perform operations on users and groups of an Azure AD. The operations are 
 performed using the v1.0 version of Microsoft Graph API. The client uses OAuth 2.0 authentication.
 
-## Building from the Source
-### Setting Up the Prerequisites
-1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
- 
-  * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
- 
-  * [OpenJDK](https://adoptopenjdk.net/)
+For more information, go to the module(s).
+- [azure.ad](aad/Module.md)
+
+## Building from the source
+### Setting up the prerequisites
+1.  Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).
+   > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed
```

Comment:
Use markdown syntax for the note, if this cannot be added into a step.

Reply:
This is added as per the [common format agreed by the team](https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit#heading=h.96g5nsmszl71). The changes which will alter that format will be added in next release cycles